deluge-mover.py

```diff
@@ -186,137 +212,124 @@ def main():
+            filtered_torrents = filter(filter_incomplete, filtered_torrents)
+        if check_fs:
+            filtered_torrents = filter(filter_not_on_cache, filtered_torrents)
+        filtered_torrents = list(filtered_torrents)
 
-            # run the mover
+        if not filtered_torrents:
             print(
-                f"[{CGREEN}init{CEND}] -> {CYELLOW}{CBOLD}Executing unRAID Mover...{CEND}\n"
+                f"\n\n[{CGREEN}deluge-mover{CEND}]: {CBOLD}no eligible torrents.\n\t\tscript completed.{CEND}\n\n"
             )
-            time.sleep(3)
             deluge_handler.call("auth.delete_session", [], 0)
             deluge_handler.session.close()
-
-            if use_mover_old:
-                system("/usr/local/sbin/mover.old start")
+            exit(0)
+        # loop through items in torrent list
+        for hash, values in filtered_torrents:
+            if check_fs:
+                save_path = to_cache_path(values["download_location"]) / values["name"]
             else:
-                system("/usr/local/sbin/mover start")
-
-            time.sleep(10)
-            print("\n\n")
-            deluge_handler = DelugeHandler()
-            auth_response = deluge_handler.call("auth.login", [deluge_password], 0)
-            time.sleep(1)
-
-            # resume all the torrents we previously paused
-            for hash, values in filtered_torrents:
-                deluge_handler.call("core.resume_torrent", [hash])
-                print(
-                    f"[{CGREEN}resume_torrent{CEND}]: {CBOLD}{values.get('name', [None])}{CEND}"
-                    f"\n\t\t  {CYELLOW}info_hash{CEND}: {hash}\n"
-                )
+                save_path = Path( values["download_location"]) / values["name"]
 
             print(
-                f"[{CGREEN}resume_summary{CEND}]: resumed {CYELLOW}{CBOLD}{len(filtered_torrents)}{CEND} torrents...\n"
-            )
-            print(
-                f"\n\n[{CGREEN}deluge-mover{CEND}]: {CBOLD}script completed.{CEND}\n\n"
+                f"[{CRED}pause_torrent{CEND}]: {CBOLD}{values['name']}{CEND}\n"
+                f"\t\t{CYELLOW}info_hash{CEND}: {hash}\n"
+                f"\t\t{CYELLOW}save_path{CEND}: {save_path}"
             )
+
+            # pause relevant torrents
+        pause_result = deluge_handler.call("core.pause_torrent", [[hash for hash, _ in filtered_torrents]])
+        if pause_result["error"]:
+            print(f"[{CRED}error pausing torrents{CEND}]: {CBOLD}{pause_result['error']}{CEND}")
+        print(
+            f"[{CRED}pause_summary{CEND}]: paused {CYELLOW}{CBOLD}{len(filtered_torrents)}{CEND} torrents...\n"
+        )
+
+        time.sleep(8)
+
+        # run the mover
+        print(
+            f"[{CGREEN}init{CEND}] -> {CYELLOW}{CBOLD}Executing unRAID Mover...{CEND}\n"
+        )
+
+        if use_mover_old:
+            system("/usr/local/sbin/mover.old start")
         else:
+            system("/usr/local/sbin/mover start")
```

Comment:
```suggestion
            if use_mover_old:
                system("/usr/local/sbin/mover.old start")
            else:
                system("/usr/local/sbin/mover start|& logger -t move")
```
this might be worth adding (the else) in case they want logging but don't want to turn on logging for normal mover operations....dunno just throwing that out there.